sonic-slave-buster/Dockerfile.j2

```diff
@@ -305,7 +305,24 @@ RUN apt-get update && apt-get install -y \
         libxml2-utils \
         xsltproc \
         python-lxml \
-        libexpat1-dev
+        libexpat1-dev \
+# For WPA supplication
+        qtbase5-dev          \
+        aspell-en            \
+        libhiredis-dev       \
+        libnl-3-dev          \
```

Comment:
We build our own libnl3 and libhiredis. Please depend on them instead of pre-install the vanilla ones in sonic-slave.